bluebutton-data-pipeline-rif/src/main/java/gov/hhs/cms/bluebutton/datapipeline/rif/model/DrugCoverageStatus.java

```diff
@@ -0,0 +1,55 @@
+package gov.hhs.cms.bluebutton.datapipeline.rif.model;
+
+/**
+ * Enumerate the possibly values for Drug Coverage Status for Part D
+ */
+public enum DrugCoverageStatus {
+	COVERED('C', "Covered by Part D"), SUPPLEMENTAL('E',
```

Comment:
Add an extra line break between the two constants so the formatter doesn't jack this up.